src/algorithm_binary.rs

```diff
@@ -52,19 +52,23 @@ impl AlgorithmBinary {
                     let range = sh.sh_offset as u32..sh.sh_offset as u32 + sh.sh_size as u32;
                     if sector.contains_range(&range) {
                         // If we found a valid section, store its contents.
-                        let data = Vec::from(&buffer[sh.sh_offset as usize..][..sh.sh_size as usize]);
+                        let data =
+                            Vec::from(&buffer[sh.sh_offset as usize..][..sh.sh_size as usize]);
                         let section = Some(Section {
                             start: sh.sh_addr as u32,
                             length: sh.sh_size as u32,
                             data,
                         });
 
                         // Make sure we store the section contents under the right name.
-                        match (&elf.shdr_strtab[sh.sh_name], FromPrimitive::from_u32(sh.sh_type)) {
+                        match (
+                            &elf.shdr_strtab[sh.sh_name],
+                            FromPrimitive::from_u32(sh.sh_type),
+                        ) {
                             CODE_SECTION_KEY => code_section = section,
                             DATA_SECTION_KEY => data_section = section,
                             BSS_SECTION_KEY => bss_section = section,
-                            _ => {},
+                            _ => {}
```

Comment:
wth rustfmt consistency pls?

Reply:
Yeah, sometimes it does strange stuff :/